packages/transcript-view/dev_tools/transcript-view-dev-options.ts

```diff
@@ -0,0 +1,316 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+import './search-results-switcher';
+
+// these are dev tools so I don't think we need coverage for them
+/* istanbul ignore */
+
+@customElement('transcript-view-dev-options')
+export default class TranscriptViewDevOptions extends LitElement {
+  @property({ type: Number }) transcriptHeight = 200;
+
+  @property({ type: Number }) topContextHeight = 50;
+
+  @property({ type: Number }) bottomContextHeight = 50;
+
+  @property({ type: Number }) currentTime = 0;
+
+  @property({ type: Boolean }) topContextVisible = true;
+
+  @property({ type: Boolean }) bottomContextVisible = true;
+
+  @property({ type: Boolean }) autoScroll = true;
+
+  @property({ type: Boolean }) showContextZones = true;
+
+  @property({ type: Boolean }) timerRunning = false;
+
+  private timer = -1;
+
+  private timerDelay = 1000;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <div>
+          <search-results-switcher
+            @searchResultIndexChanged=${this.searchResultIndexChanged}
+          ></search-results-switcher>
+        </div>
+
+        <div>
+          ${this.autoScrollTemplate}
+        </div>
+
+        <div>
+          ${this.transcriptHeightTemplate}
+        </div>
+
+        <div>
+          ${this.showContextBoxesTemplate}
+        </div>
+
+        <div>
+          ${this.topContextHeightTemplate}
+        </div>
+
+        <div>
+          ${this.bottomContextHeightTemplate}
+        </div>
+
+        <div>
+          ${this.currentTimeTemplate}
+        </div>
+      </div>
+    `;
+  }
+
+  get autoScrollTemplate(): TemplateResult {
+    return html`
+      <label>
+        <input type="checkbox" .checked=${this.autoScroll} @change=${this.changeAutoScroll} />
+        Auto Scroll
+      </label>
+    `;
+  }
+
+  get currentTimeTemplate(): TemplateResult {
+    return html`
+      <label>Current Time:</label>
```

Comment:
blocker: this label is not associated to its input.

Reply:
I'll add it.. these are all dev options so I didn't spend much time on making this file "correct"